src/createNavigationContainer.js

```diff
@@ -19,7 +19,7 @@ type NavigationContainerProps = {
   onNavigationStateChange?: (
     NavigationState,
     NavigationState,
-    NavigationAction,
+    NavigationAction
```

Comment:
Shouldn't we avoid making unnecessary changes in the code structure? 
Applies to a few other places too.